docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -6,7 +6,7 @@ Learn how to add a :doc:`custom cloud </docs/platform/concepts/byoc>` to your Ai
 
 .. topic:: Enable Bring your own cloud (BYOC)
 
-   The BYOC feature has a limited availability. From more information and the activation of the feature, contact `sales@aiven.io <mailto:sales@Aiven.io>`_.
+   The BYOC feature has a limited availability. From more information and the activation of the feature, contact `sales@Aiven.io <mailto:sales@Aiven.io>`_.
```

Comment:
Minor context update.

"...For more information, and to activate this feature, contact..."